main.js

```diff
@@ -0,0 +1,91 @@
+import $ from 'jquery'
+import './style/style.sass'
+import './style/reset.css'
+import { Todo } from './Todo.js'
+
+const inputText = $('.input-text')
+const buttonAddTodo = $('.add-item')
+const todoList = $('.todo__list')
+const filterAll = $('.filter-all')
+const filterTodo = $('.filter-todo')
+const filterDone = $('.filter-done')
+
+let todoArray = []
+
+buttonAddTodo.click(() => {
+  if (inputText.val() === '') {
+    alert('Please write any text')
+
+    return
+  }
+
+  const todo = new Todo(inputText.val())
+
+  todo.createNewTodo().appendTo(todoList)
+  
+  todoArray = todoArray.concat(todo)
+  
+  const acceptedButton = todo.getAcceptButton()
+  const deleteButton = todo.getDeleteButton()
+
+  acceptedTodo(acceptedButton, todo)
+  deleteTodo(deleteButton, todo)
+
+  inputText.val(null)
+})
+
+const acceptedTodo = (acceptedButton, todo) => {
+  acceptedButton.click(event => {
+    if (todo.isDone) {  
+      $(event.target)
+        .closest('li')
+        .removeClass('checked')
+
+      todo.unCheck.remove()
+
+      return todo.markDoneTodo().appendTo(todo.acceptedButton)
+    }
+
+    const index = todoArray.findIndex(item => item.id === todo.id)
+
+    todoArray[index].isDone = todo.isDone
```

Comment:
please add `#` to all members of the class. You shouldn't override it directly. Create a method and change isDone there